### ISSUE TITLE
If no definition exists for statement or argument, assign section of first reference

### ISSUE THREAD

**Issue by cwkit** [christianvoigt/argdown#65]
```
# Header

<ArgumentA>: Argument A

    (1) [A]: Text
    (2) [B]: Text
    ----
    (3) [C]: Text
```

```
# Header

<ArgumentB>:  Argument B

    (1) [A]: Text
    (2) [B]
    ----
    (3) [C]: Text
```

In Argument A, the Argument and all Statements are grouped in # Header. (when creating the Map)
In Argument B, Statement B is not grouped in # Header.

It seems that Statements and Arguments that have no explanatory text are always grouped outside of the respective header.

**Comment by christianvoigt** [comment]
I think I remember what you meant even if the above example is not really working (please use correct code fences for Argdown snippets). If I remember correctly the following is a minimal example for your issue: 

```argdown
# Group 1

<A>: Some description
   - <B>
```

In this case argument A belongs to Group 1 while argument B does not. This is actually the consequence of the current rules regarding group assignment (refer to the documentation for further details): Arguments belong to the group of their assigned premise-conclusion-structure or to the group in which they are first defined (with a title and and argument description). If the document contains neither a definition nor a premise conclusion structure (pcs) for the argument, but is only referred to by a title, the argument will belong to no group.

Maybe the rules have to be changed so that an argument that has neither pcs nor definition will still be assigned to the group in which it is first referenced.

Is this what you meant?

**Comment by cwkit** [comment]
I raised this issue after we discussed this specific example and your first reaction was that argdown isn't doing what it's supposed to do. You are right, it is probably best understood as a feature request because the current rules are supposed to group the statements as described. 
While it is somewhat unintuitive that a statement or an argument that is first referenced in a group is not assigned to that group, one could argue that a statement or argument without explanatory text shouldn't be named in the first place.
